packages/dotcom-ui-flags/src/__test__/server/formatFlagsJSON.spec.ts

```diff
@@ -3,14 +3,8 @@ import subject from '../../server/formatFlagsJSON'
 const fixture = Object.freeze({ foo: 1, bar: false, baz: 'qux' })
 
 describe('dotcom-ui-flags/src/server/formatFlagsJSON', () => {
-  it('returns a stringified object', () => {
```

Comment:
I've removed this since it seemed unnecessary to check the type when we do a strict assertion on an object below. Let me know what you'd prefer to do.